### ISSUE TITLE
Knocking out few classes after training the inception_v3 slim model (train_image_classifier.py)

### ISSUE THREAD

**Issue by anne1994** [tensorflow/models#2064]
In the inception_v3 slim model, the dataset is divided into training and validation sets. We train the model using the train_image_classifier.py . Is there a way to get y-train (labels intially) and y-hat-train(labels after training) to plot the confusion matrix? I want to knock out few classes in the training set based on the confusion matrix obtained from the training samples. Is it possible to do the confusion matrix for training images only ?

**Comment by shivaniag** [comment]
This question is better asked on  [StackOverflow](http://stackoverflow.com/questions/tagged/tensorflow) since it is not a  bug or feature request. There is also a larger community that reads questions there. Thanks!

**Comment by poxvoculi** [comment]
This question is better asked on  [StackOverflow](http://stackoverflow.com/questions/tagged/tensorflow) since it is not a  bug or feature request. There is also a larger community that reads questions there. Thanks!
